packages/n/networkmanager-strongswan/pspec_x86_64.xml

```diff
@@ -26,12 +26,93 @@
             <Path fileType="localedata">/usr/share/locale/de/LC_MESSAGES/NetworkManager-strongswan.mo</Path>
             <Path fileType="data">/usr/share/metainfo/NetworkManager-strongswan.metainfo.xml</Path>
         </Files>
+        <Conflicts>
+            <Package>networkmanager-libreswan</Package>
+        </Conflicts>
     </Package>
     <History>
+        <Update release="5">
+            <Date>2023-09-26</Date>
+            <Version>1.6.0</Version>
+            <Comment>Conflicting networkmanager-strongswan with libreswan, networkmanager-libreswan</Comment>
+            <Name>Maik Wöhl</Name>
+            <Email>maik.woehl@outlook.de</Email>
+        </Update>
         <Update release="4">
             <Date>2023-07-08</Date>
             <Version>1.6.0</Version>
-            <Comment>Packaging update</Comment>
+            <Comment>networkmanager-strongswan: Update to version 1.6.0
```

Comment:
I just pushed a change to the solbuild package that should prevent it from doing history generation during rebuilds. Can you please rebuild these packages one more time and then amend your commits to include the fixed pspec_x86_64.xml file? You will need to do this from a fully updated unstable system to have the most recently updated solbuild.